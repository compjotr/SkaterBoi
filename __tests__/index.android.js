jest.mock('mobx-react/native', () => require('mobx-react/custom'));
jest.dontMock('../Components/SuppMann.js');

import 'react-native';
import React from 'react';
import {shallow} from 'enzyme';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import SuppMan from '../Components/SuppMann.js';

describe('Index Android', function(){
  beforeEach(function(){
  });
  it('renders correctly', () => {
    const component = renderer.create(
      <SuppMan />
    );
    const wrapper = shallow(<SuppMan />);
    expect(wrapper.find('Button').length).toBe(1);

    expect(wrapper.find('Button').prop('children')).toBe('Halla');

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
