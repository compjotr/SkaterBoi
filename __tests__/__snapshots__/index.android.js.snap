// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F5FCFF",
      "flex": 1,
    }
  }
>
  <RCTScrollView
    dataSource={
      ListViewDataSource {
        "items": 0,
      }
    }
    enableEmptySections={true}
    renderRow={[Function]}
    renderScrollComponent={[Function]}
    style={
      Object {
        "backgroundColor": "blue",
        "flex": 1,
      }
    }
  >
    <View />
  </RCTScrollView>
  <TextInput
    onChangeText={[Function]}
    style={
      Object {
        "height": 40,
        "width": 300,
      }
    }
    value=""
  />
  <View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "backgroundColor": "red",
            "height": 60,
            "width": 210,
          }
        }
      >
        PikkPANIS
      </Text>
    </View>
  </View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    asssd
  </Text>
</View>
`;
